src/MedTrackerAPI/Features/Devices/CreateDevice.cs

```diff
@@ -0,0 +1,71 @@
+using FluentValidation;
+using MediatR;
+using MedTrackerAPI.Endpoints;
+using MedTrackerAPI.Infrastructure;
+
+namespace MedTrackerAPI.Features.Devices;
+
+public static class CreateDevice
+{
+    /*public class CreateDeviceCommandValidator : AbstractValidator<CreateDeviceCommand>
+    {
+        public CreateDeviceCommandValidator()
+        {
+            RuleFor(x => x.Description).NotEmpty();
+            RuleFor(x => x.Manufacturer).NotEmpty();
+            RuleFor(x => x.Model).NotEmpty();
+            RuleFor(x => x.PartNumber).NotEmpty();
+            RuleFor(x => x.LotNumber).NotEmpty();
+        }
+    }*/
+    
+    public sealed class Endpoint : IEndpoint
+    {
+        public void MapEndpoint(IEndpointRouteBuilder app)
+        {
+            app.MapPost("devices", async (IMediator mediator, CreateDeviceCommand command) =>
+            {
+                await mediator.Send(command);
+            }).WithTags("Devices");
+        }
+    }
+    
+    public class CreateDeviceCommand() : IRequest<IResult>
+    {
+        public string Description { get; set; }
+        public string? SerialNumber { get; set; }
+        public string  Manufacturer { get; set; }
+        public string  Model { get; set; }
+        public string  PartNumber { get; set; }
+        public string  LotNumber { get; set; }
+    }
+    
+    public class CreateDeviceCommandHandler(MedTrackerDbContext context) : IRequestHandler<CreateDeviceCommand, IResult>
+    {
+        public async Task<IResult> Handle(CreateDeviceCommand request, CancellationToken cancellationToken)
```

Comment:
Im not a fan of having the backend returning `IResult`s, as that couples the backend directly to a HTTP concept. If you need to return something back, consider something like `public int CreateDeviceCommandResponse(int Id)` which your front end then handles redirection or something.